iothub_client/samples/iothub_client_c2d_streaming_proxy_sample/iothub_client_c2d_streaming_proxy_sample.c

```diff
@@ -218,7 +225,21 @@ static void on_ws_peer_closed(void* context, uint16_t* close_code, const unsigne
 {
     (void)context;
     (void)extra_data_length;
-    (void)printf("on_ws_peer_closed (Code: %d, Data: %.*s)\r\n", *close_code, (int)extra_data_length, extra_data);
```

Comment:
Nit: we could use an inline conditional to keep the code smaller and a single `printf` call if you think it's better. (I don't have a preference.)